openapi/components/schemas/Common/CommonKycDocument.yaml

```
@@ -10,21 +11,24 @@ properties:
       - $ref: ../ResourceId.yaml
   fileId:
     deprecated: true
-    description: Linked file object id.
+    description: ID of the linked file object.
     allOf:
       - $ref: ../ResourceId.yaml
   fileIds:
-    description: >-
-      Linked file object id's.
+    description: |-
+      IDs of linked file objects.
 
-
-      Files uploaded for identity-proof must have following corresponding tags attached `['kyc', 'id-front']`, `['kyc', 'id-back']`, `['kyc', 'face-proof']` in order to be properly recognized as identity-proof files.
+      To recognized documents as identity-proof files,
+      uploaded `identity-proof` files must have the following tags attached:
+      `['kyc', 'id-front']`, `['kyc', 'id-back']`, `['kyc', 'face-proof']`.
     type: array
     items:
       allOf:
         - $ref: ../ResourceId.yaml
   documentType:
-    description: Document type submitted for validation, only identity-proof type is analyzed in an automated manner.
+    description: |-
+      Document type submitted for validation.
+      Only the `identity-proof` type is analyzed automatically.
```

This is no longer true. 
```suggestion
      Only the `identity-proof` and `address-proof` types are analyzed automatically.
```